install/terraform/open-match-build/open-match-build.tf

```diff
@@ -32,6 +32,23 @@ variable "vpc_flow_logs" {
   default     = "false"
 }
 
+variable "workload_identity_account_namespace" {
+  description = "Kubernetes namespace where the workload identity service account will take effect"
+  default     = "open-match"
+}
+
+# Change the service account name for the templates in install/helm/open-match/subcharts/open-match-stress/templates directoyr if
+# you want to modify this name.
+variable "workload_identity_account_ksa" {
+  description = "Name of the Kubernetes service account which binds to the Google service account"
+  default     = "workload-identity-ksa"
```

Comment:
This is not a meaningful name. Change it to something else like `stress-test-uploader` or something.